src/fsharp/fsi/fsi.fs

```diff
@@ -2694,7 +2724,7 @@ type FsiEvaluationSession (fsi: FsiEvaluationSessionHostConfig, argv:string[], i
     static member GetDefaultConfiguration(fsiObj:obj, useFsiAuxLib) = 
     
         let rec tryFindMember (name : string) (memberType : MemberTypes) (declaringType : Type) =
-            match declaringType.GetMember(name, memberType, BindingFlags.Instance ||| BindingFlags.Public ||| BindingFlags.NonPublic) with
+            match declaringType.GetMember(name) with //, memberType, BindingFlags.Instance ||| BindingFlags.Public ||| BindingFlags.NonPublic) with
```

Comment:
can you remove the code, instead of comment it out?
comment out code is bad on diff